testing/shairport-sync/shairport-sync.initd

```diff
@@ -0,0 +1,13 @@
+#!/sbin/openrc-run
+
+description="Implements a synchronous (multi-room-capable) AirPlay receiver"
+name="shairport-sync $SVCNAME"
+command="/usr/bin/shairport-sync"
+command_args="-d"
```

Comment:
It’s OpenRC’s job to daemonize started process, letting the process daemonize itself is a bad practice. Please remove this option and declare `command_background="yes"`.